<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1472</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soulcast Software Development Kit</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="assetTreeLayout"/>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlayGame"/>
   <addaction name="actionPlayScene"/>
   <addaction name="separator"/>
   <addaction name="actionBuildGame"/>
   <addaction name="actionExportGame"/>
   <addaction name="separator"/>
   <addaction name="actionGameSettings"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1472</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPalette_Editor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionPlayGame">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Game</string>
   </property>
   <property name="toolTip">
    <string>Run Game</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPlayScene">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/scene_32.png</normaloff>:/icons/scene_32.png</iconset>
   </property>
   <property name="text">
    <string>Run Scene</string>
   </property>
   <property name="toolTip">
    <string>Run Scene</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionBuildGame">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/build.png</normaloff>:/icons/build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="toolTip">
    <string>Build Game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExportGame">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/package.png</normaloff>:/icons/package.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export Game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionGameSettings">
   <property name="icon">
    <iconset resource="../../resources/main.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Game Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionPalette_Editor">
   <property name="text">
    <string>Palette Editor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
